setup.py

```diff
@@ -29,7 +29,7 @@
             "init_archive_retrieval=glacier_upload.initiate_job:init_archive_retrieval_command",
             "init_inventory_retrieval=glacier_upload.initiate_job:init_inventory_retrieval_command",
             "get_glacier_job_output=glacier_upload.get_job_output:get_job_output_command",
-            "abort_glacier_upload=glacier_upload.abort_upload:abort_upload_command",
+            "abort_glacier_upload=glacier_upload.manual_abort_upload:abort_upload_command",
```

Comment:
are these changes to the commands required?

Reply:
Yes, without it abort_upload doesn't work, script file named manual_abort_upload.py
or, as an alternative, we may rename script file